atuin-server/src/router.rs

```diff
@@ -72,6 +72,7 @@ pub fn router<DB: Database + Clone + Send + Sync + 'static>(
         .route("/history", post(handlers::history::add))
         .route("/history", delete(handlers::history::delete))
         .route("/user/:username", get(handlers::user::get))
+        .route("/register", delete(handlers::user::unregister))
```

Comment:
```suggestion
        .route("/me", delete(handlers::user::unregister))
```

Reply:
Perfect, I like it 😁

Reply:
sorry, I've messed this up, hence the force push